includes/classes/panel.php

```diff
@@ -0,0 +1,371 @@
+<?php
+/**
+ * Sophi Debug Bar Panel
+ *
+ * @package SophiDebugBar
+ */
+
+use SophiDebugBar\Event;
+use SophiDebugBar\Request;
+
+/**
+ * Sophi Panel class
+ */
+class SophiDebugBarPanel extends \Debug_Bar_Panel {
```

Comment:
The Debug Bar plugin doesn't support panels with namespaces (it will produce DOM element ID with backslash which will not work with their JS panel switch function). That's why using global namespace for this panel class.